osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -357,6 +357,32 @@ protected override bool OnKeyDown(KeyDownEvent e)
                 }
             }
 
+            if (e.ControlPressed)
+            {
+                if (e.ShiftPressed)
+                {
+                    switch (e.Key)
+                    {
+                        case Key.T:
```

Comment:
We have a framework-side mechanism called platform actions that might be good to use here, as tab operations are pretty well defined UI interactions overall. I can prepare a pull adding those.

Reply:
@bdach Shall we get this done framework side and try and merge this PR?

Should probably be added for the "close tab" key below as well (ctrl-w on windows; cmd-q on macOS).

Reply:
I already have added those actions, just never got round to updating this pull with them as it kept falling down the backlog.

Reply:
I've made the required changes here. @bdach can you give this a once-over whenever you have a chance?